### DISCUSSION TITLE
Lesson 6: Unable to Compile With MockV3Aggregator about brownie_fund_me

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#1050] [Q&A] [answered]
```
brownie compile
Brownie v1.17.2 - Python development framework for Ethereum

New compatible solc version available: 0.6.0
Compiling contracts...
  Solc version: 0.6.0
  Optimizer: Enabled  Runs: 200
  EVM Version: Istanbul
CompilerError: solc returned the following errors:

/home/insidtrifl/.brownie/packages/smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.6/interfaces/AggregatorV2V3Interface.sol:7:38: TypeError: Interfaces cannot inherit.
interface AggregatorV2V3Interface is AggregatorInterface, AggregatorV3Interface
                                     ^-----------------^

/home/insidtrifl/.brownie/packages/smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.6/interfaces/AggregatorV2V3Interface.sol:7:59: TypeError: Interfaces cannot inherit.
interface AggregatorV2V3Interface is AggregatorInterface, AggregatorV3Interface
                                                          ^-------------------^
```
I tried the solution from [https://githubhelp.com/PatrickAlphaC/brownie_fund_me/issues/3](url)
and changed my solc remappings from version _1.1.1_
`"@chainlink=smartcontractkit/chainlink-brownie-contracts@1.1.1"`
to version _0.2.2_
`"@chainlink=smartcontractkit/chainlink-brownie-contracts@0.2.2"`
and compiled again

but it throws this error

```
brownie compile
Brownie v1.17.2 - Python development framework for Ethereum

New compatible solc version available: 0.6.0
Compiling contracts...
  Solc version: 0.6.0
  Optimizer: Enabled  Runs: 200
  EVM Version: Istanbul
CompilerError: solc returned the following errors:

contracts/test/MockV3Aggregator.sol:4:1: ParserError: Source "/home/insidtrifl/.brownie/packages/smartcontractkit/chainlink-brownie-contracts@0.2.2/contracts/src/v0.6/interfaces/AggregatorV2V3Interface.sol" not found: File not found.
import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV2V3Interface.sol";
```

Any help would be appreciated

**Comment by cromewar** [comment, marked as answer]
Hello @InsidTrifl 
That's weird as the AggregatorV2V3Interface.sol does exists on the 0.2.2 version as you can see [here](https://github.com/smartcontractkit/chainlink-brownie-contracts/tree/0.2.2/contracts/src/v0.6/interfaces)

My recommendation on this case is:

1. use the latest version which is 0.4.0.
2. Delete your .brownie folder located in: `/home/insidtrifl/.brownie`
3. Restart all your terminals (preferably your computer) before proceed. 

Does this help you?

**Reply by ghost** [reply]
I did what you said and it still was causing problems so...
I deleted the whole fund_me folder and restarted again and it compiled on the 1.1.1 version
thanks for your help
